interface/gemv.c

```diff
@@ -89,6 +89,24 @@ static inline int get_gemv_optimal_nthreads_neoversev2(BLASLONG MN, int ncpu) {
 }
 #endif
 
+//thread throttling for dgemv
+#if defined(DYNAMIC_ARCH) || defined(NEOVERSEV1)
+static inline int get_dgemv_optimal_nthreads_neoversev1(BLASLONG MN, int ncpu) {
```

Comment:
Instead of defining a new function, I think it is cleaner to just use `get_gemv_optimal_nthreads_<uarch>`.
Inside `get_gemv_optimal_nthreads_<uarch>`, we can `#ifdef DOUBLE`.

Reply:
Yes, please keep this inside the existing function.